ironfish/src/blockchain/blockchain.ts

```diff
@@ -406,6 +406,8 @@ export class Blockchain {
 
         this.resolveOrphans(block)
 
+        this.metrics.chain_databaseSize.value = await this.db.size()
```

Comment:
Do we know what performance of this size check is like? Just want to make sure we're not adding extra time to addBlock

Reply:
I just checked the operation takes 0.031458 milliseconds, it should be negligible.